lambda-functions/jira-bot-filter-task-runner/condition-checker.js

```diff
@@ -0,0 +1,51 @@
+const moment = require('moment');
+const config = require('./config');
+
+lastHumanActivityDate = issue => {
+  const humanHistories = issue.changelog.histories.filter(history => {
+    return config.bots.indexOf(history.author.key) === -1;
+  });
+
+  const humanComments = issue.fields.comment.comments.filter(comment => {
+    return config.bots.indexOf(comment.author.key) === -1;
+  });
+
+  const allHumanActivity = humanComments.concat(humanHistories);
+  const mostRecentHumanActivityDate = allHumanActivity.reduceRight((recentDate, activity) => {
+    const activityDate = moment(activity.created);
+    return activityDate > recentDate ? activityDate : recentDate;
+  }, moment(issue.fields.created));
+
+  console.log(`Most recent human activity date for ${issue.key} is ${mostRecentHumanActivityDate}`);
+
+  return mostRecentHumanActivityDate;
+};
+
+checkCondition = (condition, issue) => {
+  let conditionStatus = false;
+  switch (condition.type) {
+    case 'lastHumanActivity':
+      const daysSinceLastHumanActivity = moment().diff(lastHumanActivityDate(issue), 'days');
+      conditionStatus = daysSinceLastHumanActivity >= condition.daysSince;
+      break;
+    case 'recentCommentTag':
```

Comment:
is their a way to use labels instead of commentTags? the nice thing about labels is they can be removed if you want to opt out of the bot watchers (or delay it)